float32 pan_angle
float32 tilt_angle
int32 pan_speed
int32 tilt_speed
